tests/support/commands.ts

```diff
@@ -57,6 +87,9 @@ Cypress.Commands.add('createRuntimeConfig', ({ type } = {}): void => {
 
   // save it
   cy.findByRole('button', { name: /save/i }).click();
+
+  // reset runtimes widget
+  cy.findByRole('tab', { name: /runtimes/i }).click();
```

Comment:
why exactly is this needed?

Reply:
Good question, I guess I added at some point and forgot to clean it up

Reply:
The failed test reminded me why :) I'm creating 2 runtimes in a row for testing kfp and airflow, and since the first step of this task is opening the runtime widget, I needed to close the widget when it's done.